controllers/Index.php

```diff
@@ -211,6 +255,15 @@ public function onDeleteObjects()
                 }
 
                 $deletedObjects = $object->delete();
+
+                /*
+                * @event pages.object.delete
```

Comment:
`@event` comment should only exist for one instance of the event being fired, it's used by API documentation generators so there is no reason to have more than one definition for a single event

Reply:
Will remove this one then.